pkg/cmd/admin/router/router.go

```diff
@@ -511,7 +514,7 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
 
 		if len(cfg.StatsPassword) == 0 {
 			cfg.StatsPassword = generateStatsPassword()
-			fmt.Fprintf(out, "password for stats user %s has been set to %s\n", cfg.StatsUsername, cfg.StatsPassword)
+			fmt.Fprintf(os.Stderr, "WARNING: password for stats user %s has been set to %s\n", cfg.StatsUsername, cfg.StatsPassword)
```

Comment:
no direct use of `os.Stderr`... if you want to separate `errout` (which I like), pass `os.Stderr` to `NewCmdRouter` and `RunCmdRouter` as an `errout` param